docs/plugins/lyrics.rst

```diff
@@ -166,6 +166,16 @@ library. Install it by typing::
 
 The backend is enabled by default.
 
+Activate Tekstowo.pl Lyrics
+----------------------
```

Comment:
The line has to be as long as the title above it:
https://travis-ci.org/github/beetbox/beets/jobs/664687146#L1104